src/carp.cc

```diff
@@ -73,24 +70,16 @@ carpInit(void)
         if (p->weight == 0)
             continue;
 
-        ++n_carp_peers;
+        rawCarpPeers.push_back(p);
 
         W += p->weight;
     }
 
-    if (n_carp_peers == 0)
+    if (rawCarpPeers.empty())
         return;
 
-    carp_peers = (CachePeer **)xcalloc(n_carp_peers, sizeof(*carp_peers));
-
     /* Build a list of the found peers and calculate hashes and load factors */
```

Comment:
Technically, we now build a selection of CARP peers earlier/above (and we are not using a _list_ for that). Let's update this stale comment:

```suggestion
    /* calculate hashes and load factors */
```

FWIW, I would prefer to remove this and similar comments that basically repeat what this code already says, but I would not recommend doing that in _this_ PR.

Reply:
Ok.